front/app/components/EventAttendanceButton/index.tsx

```diff
@@ -80,10 +88,32 @@ const EventAttendanceButton = ({ event }: EventAttendanceButtonProps) => {
       });
     } else if (user) {
       addEventAttendance({ eventId: event.id, attendeeId: user.data?.id });
+      addFollower({
```

Comment:
Feels like this should be done on the backend as there is an activity added when attendance is added.